src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests/EventHandlers/TransferRequestApprovedEventHandlerTests.cs

```diff
@@ -23,26 +23,35 @@ namespace SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests.EventHandlers
     [Parallelizable]
     public class TransferRequestApprovedEventHandlerTests
     {
-        [Test]
-        public async Task Handle_WhenHandlingTransferRequestApprovedEvent_ThenShouldFindCohortAndSetTransferApprovalProperties()
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task Handle_WhenHandlingTransferRequestApprovedEvent_ThenShouldFindCohortAndSetTransferApprovalProperties(bool autoApproval)
         {
-            var f = new TransferRequestApprovedEventHandlerTestsFixture().AddCohortToMemoryDb();
+            var f = new TransferRequestApprovedEventHandlerTestsFixture(autoApproval).AddTransferRequestAndCohortToMemoryDb();
```

Comment:
Rather than add options through the constructor, we should extend the fixture with fluent builder methods.

`AddCohortToMemoryDb` should be left alone (the name is terrible, but that is neither here nor there), and a new `AddTransferRequest` method added - _this_ method could have a parameter dictating whether auto approval or not.

Reply:
Updated.